### PR TITLE
UPSTREAM: 00000: wait for CRD discovery to be successful once before

### PR DESCRIPTION
**Pull request by mfojtik**
Working version of https://github.com/openshift/origin/pull/22146

### CODE REVIEW

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go [hunk 1]
```diff
@@ -377,6 +377,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			resourcePath = itemPath
 			resourceParams = nameParams
 		}
+		apiResource.Group = a.group.GroupVersion.Group
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go]: these are only normally set if they are different.  I suppose its not strictly wrong though.

**Comment by mfojtik** [comment]
/retest

aws flake

**Comment by mfojtik** [comment]
/retest

known flakes

**Comment by mfojtik** [comment]
@deads2k button?

/retest

**Comment by deads2k** [comment]
serial is a known flake.  This could improve our health check for a theoretical GC issue.  merging

**Pull request merged**